dataloader.py

```diff
@@ -225,14 +229,19 @@ def build_dataloader(configuration):
     )
     dataset_size = len(dataset)
     indices = list(range(dataset_size))
-    split = int(np.floor(validation_split * dataset_size))
 
+    split1 = int(np.floor(configuration.validation_ratio * dataset_size))  # 0.6
+    split2 = int(np.floor(configuration.test_ratio * dataset_size))  # 0.2
     np.random.seed(configuration.seed)
     np.random.shuffle(indices)
-    train_indices, val_indices = indices[split:], indices[:split]
+    train_indices, val_indices, test_indices = (
+        indices[0:split1],
+        indices[split1:split2],
+        indices[split2:],
+    )
```

Comment:
This is tricky to understand, maybe the following is a better option?
```
    validation_size = int(configuration.validation_ratio * dataset_size)
    test_size = int(configuration.test_ratio * dataset_size)
    np.random.seed(configuration.seed)
    np.random.shuffle(indices)
    val_indices, test_indices, train_indices = (
        indices[:validation_size],
        indices[validation_size:validation_size+test_size],
        indices[validation_size+test_size:],
    )
```
(And reduce the validation and test ratio values in the config file accordingly.)